client/landing/gutenboarding/stepper-onboarding/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { Route, Switch } from 'react-router-dom';
+import { useStepNavigation } from './hooks/use-step-navigation';
+import { Steps } from './steps';
+
+export function Stepper() {
+	const { goNext, goBack } = useStepNavigation();
+
+	return (
+		<div>
+			{ Object.values( Steps ).map( ( step ) => (
+				<Switch>
+					<Route path={ `/${ step }` } key={ step }>
```

Comment:
`step` is not guaranteed to be unique coming from Object values, right? Feels like a bit of misalignment between definition and use I think. We should maybe fix the type and extraction to guarantee uniqueness, irrespective if we don't ever really intend to add same-value steps.

But also subject to whether Step values (configs?) are [meant to remain strings](https://github.com/Automattic/wp-calypso/pull/60773/files#r798500652).